Quotient/events/roommessageevent.cpp

```diff
@@ -23,9 +23,13 @@ constexpr auto RelatesToKey = "m.relates_to"_L1;
 constexpr auto MsgTypeKey = "msgtype"_L1;
 constexpr auto FormattedBodyKey = "formatted_body"_L1;
 constexpr auto FormatKey = "format"_L1;
-constexpr auto TextTypeKey = "m.text"_L1;
-constexpr auto EmoteTypeKey = "m.emote"_L1;
-constexpr auto NoticeTypeKey = "m.notice"_L1;
+constexpr auto TextTypeId = "m.text"_L1;
+constexpr auto EmoteTypeId = "m.emote"_L1;
+constexpr auto NoticeTypeId = "m.notice"_L1;
+constexpr auto FileTypeId = "m.file"_L1;
+constexpr auto ImageTypeId = "m.file"_L1;
+constexpr auto AudioTypeId = "m.file"_L1;
+constexpr auto VideoTypeId = "m.file"_L1;
```

Comment:
These are not correct image will be m.image, video mi.video and audio m.audio

Reply:
Oh grrh, copy-pasta.